From 8e646fc8a8917fd2b829894d3097c59e735650e5 Mon Sep 17 00:00:00 2001
From: Daniele Tricoli <eriol@mornie.org>
Date: Thu, 8 Oct 2015 13:19:50 -0700
Subject: Do not use logging-clear-handlers to see all logging output and

 disable cover-min-percentage since it require python-nose (>= 1.3): this way
 it will be easier to backport python-urllib3 to Wheezy.
Forwarded: not-needed
Last-Update: 2014-7-7

Patch-Name: 04_relax_nosetests_options.patch
---
 setup.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index fbaace4..0cdcef9 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,5 +1,5 @@
 [nosetests]
-logging-clear-handlers = true
+# logging-clear-handlers = true
 with-coverage = true
 cover-package = urllib3
 cover-erase = true
